CalibTracker/SiPixelESProducers/interface/SiPixelGainCalibrationForHLTGPU.h

```diff
@@ -17,8 +17,6 @@ class SiPixelGainCalibrationForHLTGPU {
   ~SiPixelGainCalibrationForHLTGPU();
 
   const SiPixelGainForHLTonGPU *getGPUProductAsync(cuda::stream_t<>& cudaStream) const;
-  const SiPixelGainForHLTonGPU *getCPUProduct() const { return gainForHLTonHost_;}
-  const SiPixelGainCalibrationForHLT *getOriginalProduct() { return gains_;}
```

Comment:
In fact, are you suggesting to drop all of 0f57c788619c15c40083e76d05968608bd0d8561 ?